04.ls/ls.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+SPLIT_NUMBER = 3
+
+def main
+  obtained_files = obtain_files
```

Comment:
他のfilesと区別する必要もないので 

```suggestion
  files = obtain_files
```

で良いと思います。